Parsers/ASimRegistryEvent/Parsers/ASimRegistryEventNative.yaml

```diff
@@ -0,0 +1,47 @@
+Id: 9867c0fe-d890-476b-be4f-f21948b51700
+Parser:
+  Title: Registry Event ASIM parser for Microsoft Sentinel native Registry Event table
+  Version: "0.1.0"
+  LastUpdated: Nov 22 2023
+Product:
+  Name: Native
+Normalization:
+  Schema: RegistryEvent
+  Version: "0.1.2"
+References:
+  - Title: ASIM Registry Schema
+    Link: https://aka.ms/ASimRegistryEventDoc
+  - Title: ASIM
+    Link: https:/aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports normalizing the native Registry Event table (ASimRegistryEventLogs) to the ASIM Registry Event normalized schema. While the native table is ASIM compliant, the parser is needed to add capabilities, such as aliases, available only at query time.
+ParserName: ASimRegistryEventNative
+EquivalentBuiltInParser: _ASim_RegistryEvent_Native
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let parser=(disabled: bool=false) {
+    ASimRegistryEventLogs
+    | where not(disabled)
+    | project-rename
+        EventUid = _ItemId
```

Comment:
@anki-narravula The parser is running fine. You meant the error showing in the LA UI. Event with that error, we are getting expected result.
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/a55189e9-c99e-494b-a337-ba2178690ab8)

Reply:
Even with old tables, the behavior is same. May be because its a system generated field?
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/e1b2cc67-9e6d-4233-8330-5d35f3649047)